package.json

```diff
@@ -198,6 +201,7 @@
     "sass-loader": "10.1.1",
     "satisfied": "^1.1.1",
     "semver": "^6.2.0",
+    "schema-utils": "3.1.1",
```

Comment:
There dependencies (`loader-utils`, `schema-utils`) are production ones, but @Hotell suggested in one of previous PRs to add them anyway if typings packages are added.